examples/lib/defaultHtmlStepUi.js

```diff
@@ -25,6 +25,8 @@ function DefaultHtmlStepUi(_sequencer, options) {
     if (step.options && step.options.description)
       step.description = step.options.description;
 
+    var packageLink = step['docs-link'];
```

Comment:
Can we use `const` here or just write `steps['docs-link']` directly below as this variable is used only once?

Reply:
sure @HarshKhandeparkar ,it will improve code  maintainability

Reply:
Thank you!